libs/utils/utils.js

```diff
@@ -1023,28 +1024,84 @@ export async function loadMartech({
   }
 
   window.targetGlobalSettings = { bodyHidingEnabled: false };
-  loadIms().catch(() => {});
+  loadIms().catch(() => { });
 
   const { default: initMartech } = await import('../martech/martech.js');
   await initMartech({ persEnabled, persManifests, postLCP });
 
   return true;
 }
 
+/**
+ * Checks if the user is signed out based on the server timing and navigation performance.
+ *
+ * @returns {boolean} True if the user is signed out, otherwise false.
+ */
+function isSignedOut() {
```

Comment:
Isn't there another hint to detect the user signed in state? @bbythewa what was settled on, what info is coming from akamai in that regard? 🤔

Reply:
@amitbikram FYI

Reply:
Ah no, I think that's fine. Heres the context https://jira.corp.adobe.com/browse/WPS-26192